uktrade/data-hub-frontend
# Modify Datahub - Contacts with DIT content options, captions, labels etc

## Description of change

As the name of the department has changed from DIT to DBT, we will need to identify all content containing references to DIT (including both acronyms and the full department name) and update them to use the new name.

## Test instructions
Navigate Datahub front-end application into `Contacts`.

_What should I see?_
It was changed from `Department for International Trade` and `DIT` contents and acronyms into `Department for Business and Trade` and `DBT` within page title, header, options, caption, labels etc.


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
